scripts/exclude-files.txt

```diff
@@ -0,0 +1,4 @@
+src/consts.js
```

Comment:
should we move these two files out of scripts now? Not sure where though .... don't want to put too many files into root

Reply:
Good idea, maybe something like `config/link-check`?

Reply:
yep! sounds good!